code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -278,16 +278,18 @@ In most cases it makes more sense to use apply_damage() instead! And make sure t
 		UpdateDamageIcon()
 
 //Damage all limbs on your body equally
-/mob/living/carbon/human/take_overall_damage(damage, damagetype, armortype, sharp = FALSE, edge = FALSE, updating_health = FALSE, penetration)
+/mob/living/carbon/human/take_overall_damage(damage, damagetype, armortype, sharp = FALSE, edge = FALSE, updating_health = FALSE, penetration, max_limbs)
```

Comment:
If you're happy with #12122 which gives beepboops a channeled heal ability, than this max limbs change is probably not required, at least to address robots.

Reply:
I also want it since it makes medkits better so